sklearn/pipeline.py

```diff
@@ -864,11 +875,15 @@ def _validate_transformers(self):
                 continue
             if t == 'drop':
                 continue
-            if (not (hasattr(t, "fit") or hasattr(t, "fit_transform")) or not
```

Comment:
I don't see any need for change here. there's no such thin as an "intermediate step" in a FeatureUnion

Reply:
Alright, I changed that code back along with the rewritten test